tests/integration_test.rs

```diff
@@ -180,6 +210,15 @@ fn tls_client() -> Result<Client> {
         .build()
 }
 
+#[cfg(feature = "websocket")]
+fn websocket_client() -> Result<Client> {
+    Client::builder()
+        .set_host("ws://127.0.0.1:9001".to_owned())
+        .set_websocket()
+        .set_connect_retry_delay(Duration::from_secs(1))
+        .build()
+}
+
```

Comment:
I'd like to see a test that uses `wss` as well.

Reply:
Now added!